### PR TITLE
kirin: use of date verification inclusive

### PR DESCRIPTION
**Pull request by kadhikari**
A test added
timestap of gtfs-rt added in error log

### CODE REVIEW

**Comment by kadhikari** [comment]
jenkins test this please

**Comment by kadhikari** [comment]
jenkins test this please

**Comment by xlqian** [comment]
Jenkins, test this please!

File: tests/integration/gtfs_rt_test.py [hunk 1]
```diff
@@ -1218,3 +1218,111 @@ def test_gtfs_more_stops_model_builder(gtfs_rt_data_with_more_stops):
         db.session.commit()
 
         assert len(trip_updates) == 0
+
+'''
+This error message occurred many times a day for the same vehicle_journey between 1h and 2h
```
Comment (pbougue) [inline tests/integration/gtfs_rt_test.py]: please add TZ

File: tests/integration/gtfs_rt_test.py [hunk 2]
```diff
@@ -1218,3 +1218,111 @@ def test_gtfs_more_stops_model_builder(gtfs_rt_data_with_more_stops):
...
+@timestamp: 2017-12-12T01:58:44.000Z
```
Comment (pbougue) [inline tests/integration/gtfs_rt_test.py]: please add corresponding local time

File: tests/integration/gtfs_rt_test.py [hunk 3]
```diff
@@ -1218,3 +1218,111 @@ def test_gtfs_more_stops_model_builder(gtfs_rt_data_with_more_stops):
...
+impossible to calculate the circulate date (local) of vj: vehicle_journey:STS:462247-1
+Analysis:
+1. Concerns gtfs-rt which arrives between 20h and 21h localtime of the day before (1h and 2h UTC) in the morning
+with a vehicle_journey having first stop_time at mid-night localtime (5h UTC)
+
+2. since = 20171211T220000Z , until = 20171212T050000Z
+3. The first stop_time of the vehicle_journey is at 00 00 00 localtime (05h UTC) where as the gtfs-rt arrives after
+ 1h UTC -> GTFS-RT has an information on a vehicle journey with departure after 4 hours !!!
```
Comment (pbougue) [inline tests/integration/gtfs_rt_test.py]: with departure 4 hours `in the future`

File: tests/integration/gtfs_rt_test.py [hunk 4]
```diff
@@ -1218,3 +1218,111 @@ def test_gtfs_more_stops_model_builder(gtfs_rt_data_with_more_stops):
...
+
+4. Filter in the code:
+ since_local = 20171211T170000 -05:00, until_local = 20171212T000000 -05:00
+
+ since_local <= date(since_local) + 00 00 00 <= until_local -> false
```
Comment (pbougue) [inline tests/integration/gtfs_rt_test.py]: I don't understand that (why it's false, and what's the meaning of `+ 00 00 00`)

File: kirin/gtfs_rt/model_maker.py [hunk 5]
```diff
@@ -111,7 +111,7 @@ def _get_stop_code(self, nav_stop):
             if c['type'] == self.stop_code_key:
                 return c['value']
 
-    def _make_trip_updates(self, input_trip_update, data_time):
+    def _make_trip_updates(self, input_trip_update, data_time, time_stamp):
```
Comment (pbougue) [inline kirin/gtfs_rt/model_maker.py]: not super fan of a new attribute of a value that's the same as data_time

**Pull request merged**